### DISCUSSION TITLE
Celsius vs Fahrenheit in Climate Mode

### DISCUSSION THREAD

**Discussion by Mermarky** [basbruss/adaptive-cover#349] [Q&A]
All my devices for outside and inside temperature report in Fahrenheit. Am I correct that there is no option in Climate Mode to use Fahrenheit?

**Comment by asiertxo78** [comment]
I don't think it matters if the values are given in Celsius or Fahrenheit, you just need to set a value between 0 and 86, as there are no units specified anywhere.
I hope this helps.
